Lombiq.BaseTheme/Assets/Styles/abstracts/_mixins.scss

```diff
@@ -221,7 +225,9 @@
 
     @include media-breakpoint-only(lg) { #{$property}: calc(#{$multiplier} * var(--gutter-large)); }
 
-    @include media-breakpoint-up(xl) { #{$property}: calc(#{$multiplier} * var(--gutter-x-large)); }
+    @include media-breakpoint-only(xl) { #{$property}: calc(#{$multiplier} * var(--gutter-x-large)); }
+
+    @include media-breakpoint-up(xl) { #{$property}: calc(#{$multiplier} * var(--gutter-xx-large)); }
```

Comment:
Shouldn't this be `media-breakpoint-up(xxl)` too like on the other places? (Instead of `xl`).